scout/server/links.py

```diff
@@ -625,6 +626,12 @@ def mitomap_link(variant_obj):
     return url_template.format(this=variant_obj)
 
 
+def hmtvar_link(variant_obj):
+    """Compose a link to a variant in HmtVar"""
+    url_template = "https://www.hmtvar.uniba.it/varCard/{id}"
```

Comment:
Works perfectly: 
![image](https://user-images.githubusercontent.com/28093618/116216425-e5cf8b00-a748-11eb-9b66-ef45f09296ed.png)

Reply:
An improvement would be to link to the actual position of the variant in HmtVar: https://www.hmtvar.uniba.it/varCard/39192

Reply:
Hmm, I don't see the difference? At least that was what I was trying to achieve, and it works here. What does the link look like for you?

Reply:
I agree, It doesn't really matter. I just noticed that the link for this demo variant:

![image](https://user-images.githubusercontent.com/28093618/116353262-baa37500-a7f6-11eb-9cea-a12ec01db10c.png)

leads to another variant on HmtVar: https://www.hmtvar.uniba.it/varCard/12345, probably because you used a random number (12345) as hmtvar ID. The id for that specific variant is another, but it's fine, really, since it's just test data.

Reply:
Yes sorry, I was lazy and just put a random number there...